### PR TITLE
Add io.github.cosmic_utils.Examine

### PR DESCRIPTION
**Pull request by sungsphinx**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. Examine is an app built using libcosmic to view system and hardware information, right now it lets you view info about your distribution, processor, pci devices and usbs.
  

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

The `--device=all` permission is to access usb devices so the `lsusb` command works, which the app needs for the USBs page.

Linter PR for `xdg-config/cosmic:ro`: https://github.com/flathub-infra/flatpak-builder-lint/pull/503

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.cosmic_utils.Examine.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    "url": "https://github.com/casey/just/releases/download/1.25.2/just-1.25.2-aarch64-unknown-linux-musl.tar.gz",
+                    "sha256": "049d624255d150f18b0f3a1257a163366b0dc22e16526214773903125145e515",
+                    "only_arches": [
+                        "aarch64"
+                    ]
+                }
+            ]
+        },
+	{
+            "name": "libusb",
```
Comment (hfiguiere) [inline io.github.cosmic_utils.Examine.json:62]: please use the shared-modules

File: io.github.cosmic_utils.Examine.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            "CARGO_HOME": "/run/build/examine/cargo",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+        }
+    },
+    "modules": [
+        {
+            "name": "just",
```
Comment (bbhtt) [inline io.github.cosmic_utils.Examine.json:33]: this is in the baseapp

File: io.github.cosmic_utils.Examine.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+                    "url": "https://github.com/casey/just/releases/download/1.25.2/just-1.25.2-aarch64-unknown-linux-musl.tar.gz",
+                    "sha256": "049d624255d150f18b0f3a1257a163366b0dc22e16526214773903125145e515",
+                    "only_arches": [
+                        "aarch64"
+                    ]
+                }
+            ]
+        },
+	"shared-modules/libusb/libusb.json",
+        {
```
Comment (bbhtt) [inline io.github.cosmic_utils.Examine.json:62]:
```suggestion
        },
	"shared-modules/libusb/libusb.json",
        {
```
```suggestion
        },
	      "shared-modules/libusb/libusb.json",
        {
```

File: io.github.cosmic_utils.Examine.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            "name": "usbutils",
```
Comment (bbhtt) [inline io.github.cosmic_utils.Examine.json:63]: can you fix the indentation here?

image

**Comment by bbhtt** [comment]
bot, build io.github.cosmic_utils.Examine

**Pull request closed without merging**